internal/cluster/distribution/eks/eksworkflow/workflow_update_cluster.go

```diff
@@ -111,59 +109,6 @@ func (w UpdateClusterWorkflow) Execute(ctx workflow.Context, input UpdateCluster
 		}
 	}
 
-	// check add-on are enabled and K8s version is >= 1.18
```

Comment:
**Required**: `enableAddons` **MUST** also be removed from the workflow.

Reply:
removed